src/tir/schedule/trace.cc

```diff
@@ -476,6 +476,8 @@ TVM_STATIC_IR_FUNCTOR(ReprPrinter, vtable)
     .set_dispatch<TraceNode>([](const ObjectRef& obj, ReprPrinter* p) {
       const auto* self = obj.as<TraceNode>();
       ICHECK_NOTNULL(self);
+      p->stream << "# from tvm import tir" << std::endl;
+      p->stream << "def apply_trace(sch: tir.Schedule) -> None:" << std::endl;
```

Comment:
Just a minor suggestion that frequently calling IO flushing via `std::endl` for shorter strings might be slow. So putting a `std::flush` in the end and keep using `<< '\n'` for newlines other than `<< std::endl` in previous places is usually more efficient.

Reply:
Sure. Usually I don't particularly favor performance when implementing features that are not on critical paths, but indeed it's good advice. Updated!